assets/src/edit-story/components/library/panes/shared/chipGroup/useHandleRowVisibility.js

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Copied from the adjacent `pillGroup` folder

Reply:
Could these two (pills / chips) not use the same `useHandleRowVisibility` and `useExpandAnimation`? Which differences are in these copied files compared to the original, could these perhaps be implemented via props passed to it instead?

Or will the `pillGroup` not be in use anymore at all?

Reply:
I think the pill group won't be used at all eventually but it would probably be better for them to use the same hooks! They're pretty much the same other than some constants

But that can be generalized for sure 👍